src/outcomes-list/partial-bold.js

```diff
@@ -69,6 +69,11 @@ export class PartialBold extends mixinBehaviors(
 			return acc;
 		}, []);
 	}
+
+	_getScreenReaderText(parsedContent) {
+		const data = parsedContent.map((item) => item.data);

```

Comment:
If you use `reduce`, you only need to iterate once. (Then join is not necessary)

```suggestion
		const data = parsedContent.reduce((acc, cur) => return acc + cur.data, '');

```